games/godot/Portfile

```diff
@@ -190,8 +189,8 @@ proc get_major {version} { return [lindex [split ${version} .] 0] }
 
 set clang_versions {5.0 6.0 7.0 8.0 9.0 10 11}
```

Comment:
Clang 12 and 13 are now available. If supported they could be added to this list.

Reply:
Done! :smile:

(I did verify that the build completes successfully using Clang 12 and 13)

Reply:
Thanks. Can you though please squash the two commits into ine.

Reply:
Oops, I forgot to do that, sorry. Should now be squashed.